EventsExpress/ClientApp/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please modify `.gitignore` to ignore .vscode folders and do not iclude such in sources.

Reply:
Accidentally added `VSCode` files have been removed and the `.gitignore` file has been changed.